server/service/testing_utils.go

```diff
@@ -190,7 +191,11 @@ func RunServerForTestsWithDS(t *testing.T, ds fleet.Datastore, opts ...*TestServ
 	if len(opts) > 0 && opts[0].Lq != nil {
 		lq = opts[0].Lq
 	}
-	svc := newTestService(t, ds, rs, lq, opts...)
+	cfg := config.TestConfig()
```

Comment:
Before the change, it was using a zero value for the config (`config.FleetConfig{}`), but if the tests pass with the test config that's fine (CI hasn't finished as I'm reviewing this).

Reply:
the tests passed locally 🎉 , so I'm hoping they will pass in CI too. Thanks for validating the assumption that it was OK to use the test config.